plone/app/layout/globals/context.py

```diff
@@ -163,7 +169,14 @@ def folder(self):
 
     @memoize
     def is_folderish(self):
-        return bool(getattr(aq_base(aq_inner(self.context)), 'isPrincipiaFolderish', False))
+        cond1 = bool(
+            getattr(aq_base(aq_inner(self.context)),
+                    'isPrincipiaFolderish', False))
+        cond2 = False
+        if HAS_PAC:
+            if ICollection.providedBy(self.context):
+                cond2 = True
+        return cond1 or cond2
```

Comment:
@kiorky Can you explain how eea and other addons rely on collections being folderish even though they are in fact not? I don't think this hack is a great solution, instead eea should use 'results' if it uses new collections to access the content.